shared/src/bundles/resumes/content/education/education-dto.type.ts

```diff
@@ -0,0 +1,16 @@
+type Education = {
+    id: string;
+    resumeId: string;
+    institution: string;
+    degree: string;
+    startDate: string;
+    endDate: string;
+    currentlyStudying: boolean;
```

Comment:
There was a comment about the prefix. But it would be better to come up with a more suitable name